v2/trace/aws/inject.go

```diff
@@ -77,18 +77,16 @@ func getTraceAttributesFromContext(ctx context.Context) map[string]string {
 	if span := oc_trace.FromContext(ctx); span != nil {
 		spanCtx := span.SpanContext()
 
-		attributes[b3TraceHeader] = hex.EncodeToString(spanCtx.TraceID[:])
```

Comment:
Should we no longer provide B3 headers?

Reply:
We are trying to minimize the amount of message attributes in use (due to the limitation of max 10 messages attributes for SQS). Going from 4 to 2.
The aws wrapper still converts the b3 headers to dd, and the only subscriber/receiver of these headers we have currently is datadog. We don't have any lambdas currently running opencensus (what I know of), if that is something we want in the future, we need to add a subscriber/reciever for creating opencensus spans and maybe re-add this header (and maybe rewrite this solution to combine all headers in just one message attribute, e.g json).

Reply:
Ok!